python/codepipeline-docker-build/app.py

```diff
@@ -0,0 +1,26 @@
+from aws_cdk import (
+    core,
+)
+
+from Base import Base
+from Pipeline import Pipeline
+
+
+# using props to pass in objects between stacks
+class Props():
+    def __init__(self):
+        self.namespace = 'cdk-example-pipeline'
+        self.region = 'us-east-1'
+
+
+props = Props()
+app = core.App()
+
+# stack for ecr, bucket, codebuild
+base = Base(app, f"{props.namespace}-base", props, )
+props = base.outputs
+
+# pipeline stack
+pipeline = Pipeline(app, f"{props.namespace}-pipeline", props)
```

Comment:
I would skip modifying `props`. Having the same variable mean different things in the program is just more overhead to me. Just pass `base.outputs` here.

Reply:
I like the idea of passing around an object that is independent of a single stack. If we had a third stack here that needed objects from stack 1 and 2, I think this way works better - unless you were thinking of something else?